docs/installation/dns_setup.md

```diff
@@ -1,157 +1,201 @@
 ---
 sidebar_label: "DNS setup"
 sidebar_position: 3
-title: ""
+title: "DNS setup"
+description: How to setup DNS for Epinio and associated issues.
+keywords: [epinio, kubernetes, k8s, DNS]
 ---
 
-#  DNS setup
+During an [Epinio installation](install_epinio.md),
+a "system" domain must be specified in the `global.domain` helm field.
+This field is used to access an Epinio API server and to create default routes for the deployed applications.
```

Comment:
This value is used for creating an ingress resource for accessing Epinio API server and Epinio webUI...

Reply:
Actually "Epinio API server and an Epinio WebUI" are using the same ingress named `epinio` (listening on **epinio.**`test.example.com`) but I believe it is worth to mention it. 

Now I noticed another ingress called `dex` (listening on **auth.**`test.example.com`) used for `Log in with Auth Provider` on Epinio login page.

I like the first option enhanced by dex:

During an Epinio installation a system domain must be specified in the 'global.domain' helm field. This field is used to create an ingress resource for access to an Epinio API server, an Epinio WebUI, a DeX and to create default routes for the deployed applications.